resources/tools/mdsd/devops/pipeline/stages/impl/dockerized/modules/Build/Build.groovy

```diff
@@ -0,0 +1,13 @@
+MPLModule('Setup Container')
+
+CFG = updateConfiguration()
+
+if (!CFG.dockerBuildImage) {
+    error('Configuration error. Docker Image is expected to be set by specializing Module implementation.')
+}
+
+// run maven build in docker container
```

Comment:
Comment is wrong because it is no longer restricted to Maven.

Reply:
Yes